tsdb/chunkenc/chunk.go

```diff
@@ -162,5 +161,5 @@ func FromData(e Encoding, d []byte) (Chunk, error) {
 	case EncXOR:
 		return &XORChunk{b: bstream{count: 0, stream: d}}, nil
 	}
-	return nil, fmt.Errorf("unknown chunk encoding: %d", e)
+	return nil, errors.Errorf("invalid encoding %q", e)
```

Comment:
I think it's rather nice to know that it's chunk encoding being invalid. Do you think it's not an important detail? (:

Reply:
It's a good question. I think that error should contain important detail.

Reply:
sure, but let's add `chunk` detail?>

Reply:
updated.